lib/Doctrine/DBAL/Cache/ArrayStatement.php

```diff
@@ -48,7 +49,7 @@ public function __construct(array $data)
      */
     public function closeCursor() : void
     {
-        unset($this->data);
+        $this->data = null;
```

Comment:
I was adding tests for ArrayStatement and this seemed wrong since it can result in PHP errors if you call other public methods after `closeCursor()`

Reply:
Makes perfect sense.